docs/data-guidelines/api.md

```diff
@@ -2,6 +2,49 @@
 
 This file contains guidelines that are specific to the web API features (`api/`).
 
+
+## Adding features and subfeatures
+
+A json file should be created for every _top level feature_ that has a released public implementation on one of the supported set of browsers (and not before).
```

Comment:
The "and not before" here should come with a lint I think, and I think running that lint will reveal that more things are removed by this rule than perhaps we really want to see removed.

But if we can sort through that, I think that we should add things to BCD when they're in a beta browser release, to not have to wait until stable before documenting things, if someone is willing to do the work earlier.